install/tests/Makefile

```diff
@@ -7,26 +7,10 @@ TOPDIR=$(shell pwd)
 
```

Comment:
**question:** We're silencing Make's default behavior to print commands before running them. Could this make tests more difficult to debug? Does this improve the overall user experience? Who benefits the most/the least from silencing that output?

Reply:
I think silencing the commands itself makes it much easier to debug since we can actually see on the error messages. With this change the logs become much more readable. Alongside the user who will be getting kubeconfig will have a smoother experience.